pkg/meta/openfile.go

```diff
@@ -222,6 +226,7 @@ func (o *openfiles) CacheChunk(ino Ino, indx uint32, cs []Slice) {
 		}
 		of.chunks[indx] = cs
 	}
+	of.lastCheck = time.Now().Unix()
```

Comment:
The `lastCheck` is used to indicate the check time of attribute, so should not be updated here.